runtime/config-deployer-service/ballerina/artifact-deployer-api_service.bal

```diff
@@ -20,7 +20,7 @@ isolated service /api/deployer on ep0 {
     # AcceptedString (API undeployed successfully)
     # BadRequestError (Bad Request. Invalid request or validation error.)
     # InternalServerErrorError (Internal Server Error.)
-    isolated resource function post apis/undeploy(string apiId) returns AcceptedString|BadRequestError|InternalServerErrorError|commons:APKError {
+    isolated resource function post apis/undeploy(string apiId) returns AcceptedString|BadRequestError|InternalServerErrorError|commons:APKError|error {
```

Comment:
Any idea why we need error in the returns here? without this I am getting a compilation error

Reply:
With ballerina `2201.7.0` I could compile the latest master code without any issue